js/siteorigin-panels/view/live-editor.js

```diff
@@ -32,7 +32,47 @@ module.exports = Backbone.View.extend( {
 		    // Scroll to the correct position
 		    thisView.$el.find( '.so-preview iframe' ).contents().scrollTop( thisView.previewScrollTop );
 
+		    var $$ = $(this ),
+			    ifc = $$.contents();
```

Comment:
How about `$braam'sIframeContainer`? Yeah, you're right. I remember actually forgetting what that was.